orangecontrib/text/datasets/url.tab

```diff
@@ -0,0 +1,19 @@
+topic	url_case_study	url

```

Comment:
What's the difference between `url_case_study` and `url`? Or why are they stored in the same row?